docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+</Tabs>
+
+Replace the following fields with values from your own cluster.
+
+<ScopedBlock scope={["cloud"]}>
+
+- `token` is the token output by the `tctl bots add` command or the name of your IAM method token.
+- `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
+- `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
+- `auth-server` is the address of your Teleport Cloud Proxy Server, for example `example.teleport.sh:443`.
+- `certificate-ttl` is the option that will result in creation of a host certificate with Machine ID. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
+
+</ScopedBlock>
+<ScopedBlock scope={["oss","enterprise"]}>
+
+- `token` is the token output by the `tctl bots add` command or the name of your IAM method token.
+- `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
+- `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
+- `certificate-ttl` is the option that will result creation of a host certificate with Machine ID. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
+- `auth-server` is the address of your Teleport Auth Server, for example
+  `auth.example.com:3025`. If your Machine ID host is in a different network
+  than your Auth Server, use the public web address of your Proxy Service
+  instead (e.g., `auth.example.com:443`).
+
+
+</ScopedBlock>
+
+Using a text editor of your choice, write the output generated by the `tbot configure` command to a configuration YAML file, in this example we'll be using `tbot.yaml`.
+
+Once complete, the Machine ID configuration will still require a destination directory to be manually referenced within the configuration file. The destination directory is where Machine ID will write user certificates
+which can be used by application and tools to start Machine ID. In the configuration yaml `tbot.yaml`, this configuration will additionally be used to designate the **principal**,
+in this case the Domain Name that we'll be applying to the tbot configuration to access a Node.
+The new configuration YAML will require a `destination` field and `ssh_host_cert` subfield that resembles the following, replacing `nodename.my.domain.com` with your own required domain name:
```

Comment:
```suggestion
The new configuration YAML will require a `destination` field and `ssh_host_cert` subfield that resembles the following, replacing `nodename.my.domain.com` with your own required domain names or IP addresses:
```

Might be worth mentioning IP addresses are valid principals, since I'm sure some deployments won't have DNS set up at all.